assets/css/less/front-end/elements/buttons.less

```diff
@@ -0,0 +1,112 @@
+/* stylelint-disable no-descending-specificity */
+
+/**
+ * Buttons Element Styles
+ *
+ * This file includes the styles for the buttons element in the plugin.
+ */
+
+body .wpbdp-page,
+body .wpbdp-single,
+body .wpbdp-excerpt {
+
+	input[type="submit"],
+	input.submit,
+	a.wpbdp-button,
+	.wpbdp-button,
+	a.reset,
+	#change-plan-link {
+		display: inline-flex;
+		align-items: center;
+		justify-content: center;
+		cursor: pointer;
+		white-space: nowrap;
```

Comment:
Looks like we're setting the button padding for some themes like
`.wpbdp-wp-theme-twentytwentytwo .wpbdp-page .wpbdp-main-links .button`

It looks like we need to add this for the 2023 theme too?